save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/SaveAgent.kt

```diff
@@ -121,6 +141,57 @@ class SaveAgent(private val config: AgentConfiguration,
         coroutineScope.cancel()
     }
 
+    // a temporary workaround for python integration
+    private fun executeAdditionallySetup(targetDirectory: Path, additionalFiles: List<FileKey>) = runCatching {
+        additionalFiles
+            .singleOrNull { it.name == "setup.sh" }
+            ?.let { fileKey ->
+                val targetFile = targetDirectory / fileKey.name
+                logDebugCustom("Additionally setup of evaluated tool by $targetFile")
+                val setupResult = ProcessBuilder(true, FileSystem.SYSTEM)
+                    .exec(
+                        targetFile.toString(),
```

Comment:
Shouldn't it be `./$targetFile`? Or maybe we should call shell ourselves in case their script doesn't specify any interpreter?

Reply:
I see that `save-cli` is called by `./script-name` -- will fix it
but I don't think that we should call a shell. I think it's responsibility of script's provider